### PR TITLE
Review SDK commands for 2.1

### LINKED ISSUE #4627: [Feature] Revisit and revise .NET Core SDK Commands
# Revisit and revise .NET Core SDK Commands

Revise .NET Core SDK Commands for .NET Core 2.1.  

edit by @mairaw: removed TOC and content type from still needed. Most of the SDK commands already exist under https://docs.microsoft.com/en-us/dotnet/core/tools/.

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #4627 
Related to #3177

### CODE REVIEW

File: docs/core/tools/dotnet-migrate.md [hunk 1]
```diff
@@ -15,24 +15,27 @@ ms.date: 08/14/2017
 
 ## Synopsis
 
-`dotnet migrate [<SOLUTION_FILE|PROJECT_DIR>] [-t|--template-file] [-v|--sdk-package-version] [-x|--xproj-file] [-s|--skip-project-references] [-r|--report-file] [--format-report-file-json] [--skip-backup] [-h|--help]`
+```
```
Comment (BillWagner) [inline docs/core/tools/dotnet-migrate.md]: Should this be ```console?

File: docs/core/tools/dotnet-migrate.md [hunk 2]
```diff
@@ -95,4 +98,4 @@ Migrate all projects that *global.json* file includes:
 
 Migrate only the current project and no project-to-project (P2P) dependencies. Also, use a specific SDK version:
 
-`dotnet migrate -s -v 1.0.0-preview4`
+`dotnet migrate -s -v 1.0.0-preview4`
```
Comment (BillWagner) [inline docs/core/tools/dotnet-migrate.md]: Should this be code-fenced as ```console?

**Review by BillWagner — APPROVED** [review]
Overall, this looks great @mairaw 

I had a couple small nits on code fencing, which you can take or ignore based on context.

Go ahead and :shipit: when ready.

File: docs/core/tools/dotnet-migrate.md (hunk 1 above)
Comment (mairaw) [reply docs/core/tools/dotnet-migrate.md]: I haven't been using anything for the CLI commands. I talked to Curt and they will create a dotnetcli for us, but it'll take some time.

**Comment by JRAlexander** [comment]
LGTM.

**Pull request merged**